### PR TITLE
[documentation] recenter logo on README

### PR DESCRIPTION
**Pull request by danbugs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
+Hyperlight is a library for creating _micro virtual machines_  — or _sandboxes_ — specifically optimized for securely
+running untrusted code with minimal impact. It supports both Windows and Linux,
+utilizing [Windows Hypervisor Platform](https://docs.microsoft.com/en-us/virtualization/api/#windows-hypervisor-platform)
+on Windows, and either Microsoft Hypervisor (mshv) or [KVM](https://linux-kvm.org/page/Main_Page) on Linux.
 
-These micro VMs operate without a kernel or operating system, keeping overhead low. Instead, guests are built specifically for Hyperlight using the Hyperlight Guest library, which provides a controlled set of APIs that facilitate interaction between host and guest:
+These micro VMs operate without a kernel or operating system, keeping overhead low. Instead, guests are built
```
Comment (ludfjig) [inline README.md:19]: nit: Are the new manual linebreaks intended? I'm no expert on markdown, and I'm not sure if you should linebreak manually or not :D

**Review by ludfjig — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (danbugs) [reply README.md:19]: They are not—though, a lot of people add them. My editor has automatic soft-wrapping so it's a non-issue for me 🤷 . Doesn't hurt to leave them, though.

Comment (danbugs) [reply README.md:19]: Oh, I just realized there were a lot of changes to the README besides what I added. I think this was applied by auto-formatting.

**Review by simongdavies — APPROVED** [review]

**Pull request merged**